packages/environment/package.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "@sveltejs/environment",
+	"version": "1.0.0-next.1",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/sveltejs/kit",
+		"directory": "packages/environment"
+	},
+	"license": "MIT",
+	"homepage": "https://kit.svelte.dev",
```

Comment:
create readme in this package and point homepage there? esp. if we want the community to use this package standalone instead of via kit.

Reply:
done